app/components/article-content.js

```diff
@@ -112,10 +112,20 @@ export default Component.extend(
 				let lightboxModel;
 
 				if (gallery) {
-					lightboxModel = JSON.parse(gallery.getAttribute('data-attrs'));
-					lightboxModel.galleryRef = parseInt(figure.getAttribute('data-ref'), 10);
+					try {
+						lightboxModel = JSON.parse(gallery.getAttribute('data-attrs'));
```

Comment:
sorry to be picky but
this looks almost exactly like code in 'else'
maybe we could extract it to a method?
`getLightboxModel(elem); -> object` or something like that?